src/components/process-accordion/process-accordion.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import {
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+} from '@reach/accordion';
+import '@reach/accordion/styles.css';
+import styled from 'styled-components';
+import { CaptionText } from '../typography/typography';
+import { useTranslation } from 'gatsby-plugin-react-i18next';
+import { LaunchingRocket } from '../icons/launching-rocket';
+import { Circle } from '../icons/circle';
+import { FlyingRocket } from '../icons/flying-rocket';
+import { FlagOnEarth } from '../icons/flag-on-earth';
+import { Satellite } from '../icons/satellite';
+
+const ProcessCaption = styled(CaptionText)`
+  font-size: 16px;
+  margin: 16px;
+`;
+
+const StyledAccordion = styled(Accordion)`
+  display: relative;
+`;
+
+const StyledAccordionPanel = styled(AccordionPanel)`
+  padding: 16px;
+  padding-top: 8px;
+  font-size: 16px;
+  line-height: 150%;
+  white-space: pre-line;
+
+  > svg {
+    display: block;
+    margin-bottom: 16px;
+  }
+`;
+
+const StyledAccordionButton = styled(AccordionButton)`
+  background: none;
+  border: none;
+  padding: 0px;
+
+  display: relative;
+  width: 100%;
+  cursor: pointer;
+  border-top: 1px solid rgba(255, 255, 255, 0.1);
+  text-align: left;
+`;
+
+const StyledAccordionItem = styled(AccordionItem)`
+  &:hover {
+    background: linear-gradient(
+      180deg,
+      rgba(77, 121, 255, 0.1) 0%,
+      rgba(77, 121, 255, 0) 100%
+    );
+  }
+
+  &:last-child {
+    border-bottom: 1px solid rgba(255, 255, 255, 0.1);
+  }
+
+  &[data-state='open'] {
+    position: relative;
+    &::after {
+      content: url("data:image/svg+xml;utf8,<svg width='12' height='8' viewBox='0 0 12 8' fill='none' xmlns='http://www.w3.org/2000/svg'>
+      <path d='M10 6L6 2L2 6' stroke='%23668CFF' stroke-width='2.5' stroke-linecap='round'/></svg>");
+      position: absolute;
+      z-index: -1;
+      top: 15px;
+      right: 24px;
+    }
+  }
+
+  &[data-state='collapsed'] {
+    position: relative;
+    &::after {
+      content: url("data:image/svg+xml;utf8,<svg width='12' height='8' viewBox='0 0 12 8' fill='none' xmlns='http://www.w3.org/2000/svg'>
+      <path d='M10 2L6 6L2 2' stroke='%23668CFF' stroke-width='2.5' stroke-linecap='round'/></svg>");
+      position: absolute;
+      z-index: -1;
+      top: 15px;
+      right: 24px;
+    }
+  }
+`;
+
+export const ProcessAccordion = () => {
+  const { t } = useTranslation();
+  return (
+    <StyledAccordion collapsible multiple>
+      <StyledAccordionItem>
+        <StyledAccordionButton>
+          <ProcessCaption>
+            {t('career.process.application.title')}
+          </ProcessCaption>
+        </StyledAccordionButton>
+        <StyledAccordionPanel>
+          <LaunchingRocket />
+          {t('career.process.application.text')}
+        </StyledAccordionPanel>
+      </StyledAccordionItem>
```

Comment:
I would have expected something like this:

```
<OurAccordionItem title={} illustration={LaunchingRocket}>
 {/** content **/}
</Accordion>
```

Where `OurAccordionItem` is our custom component bundling everything that is repeated. This is some code golfing idea here but some obvious one I think. Because it would also solve the issue with the toggle icons being inline in css here ( see other comment)